src/main/java/sdmExam/Board.java

```diff
@@ -5,25 +5,41 @@
 import java.util.Optional;
 
 public class Board {
-    private List<Cell> cells = new ArrayList<>();
+    private final List<Intersection> intersections = new ArrayList<>();
+    private Mark lowerAndUpperEdgesColor = Mark.BLACK;
+    private Mark leftAndRightEdgesColor = Mark.WHITE;
 
     public Board() {
-        // Board is 13x13 squares, but we can use edges and corners to mark so it is 14x14 cells
-        //TODO: think about a way to model the borders the board
-        for (int row = 0; row < 14; row++) {
```

Comment:
we may put a typedef variable for the 14 number, let's say an external private variable. In order to avoid "magic number".